src/main/filtered-resources/etc/kafka-event-mirrorer.init

```diff
@@ -25,9 +25,10 @@ SYSCONFDIR="@SYSCONFDIR@"
 STOP_RETRIES=3
 STOP_WAIT=10
 
-RUNAS="root"
+RUNAS="@RUNAS@"
 PIDFILE="$APP_PREFIX/run/$NAME.pid"
-LOGFILE="@LOGFILE@"
+LOGDIR="/var/log/kem/"
+LOGFILE="$LOGDIR/kem.log"
```

Comment:
Aren't and stdout/stderr going to the same logfile that logback is using now?

Reply:
It's not coupled in any way. The init script is using $LOGFILE for STDOUT/STDERR.
The Java process log is configured in the logback.xml and both configurations are set to the same '/var/log/kem.log' value.
Did you want them more tightly coupled?